src/utils/verifier.js

```diff
@@ -0,0 +1,39 @@
+const { httpRequest } = require('../utils')
+
+module.exports = {
+  /**
+   * Verifies that the gist contains the given muportDID and returns the users github handle.
+   * Throws an error otherwise.
+   *
+   * @param     {String}            did                     The muport DID of the user
+   * @param     {Object}            gistUrl                 URL of the proof
+   * @return    {String}                                    The github handle of the user
+   */
+  verifyGithub: async (did, gistUrl) => {
+    console.log({did, gistUrl});
+    if (!gistUrl || gistUrl.trim() === "") {
+      throw new Error("The proof of your Github is not available");
+    }
+    if (!did || did.trim() === "") {
+      throw new Error("DID parameter not provided");
+    }
+    let gistFileContent = await (await fetch(gistUrl)).text();
```

Comment:
Please use httpRequest (imported above) for this. The main reason is to be consistent with how we do http requests.